src/vs/workbench/parts/debug/electron-browser/repl.ts

```diff
@@ -375,3 +390,26 @@ CommonEditorRegistry.registerEditorCommand(new SuggestCommand({
 		primary: KeyCode.RightArrow
 	}
 }));
+
+@editorAction
+class ReplCopyAllAction extends EditorAction {
+
+	constructor() {
+		super({
+			id: 'repl.action.copyall',
+			label: nls.localize('actions.repl.copyall', "Debug Copy All"),
```

Comment:
I would prefer the title to be "Debug: Console Copy All", that way it nicely aligns with the other debug actions and has a more clear name